src/resolvers/claims.ts

```diff
@@ -1,23 +1,34 @@
-import {Claim, ClaimOutcome, ClaimResolvers, ClaimStatus, QueryResolvers} from "../generated/graphql";
-import {ClaimDto, ClaimOutcomeDto, ClaimStatusDto} from "../api/upstreams/claimService";
+import {Claim, ClaimOutcome, ClaimResolvers, ClaimStatus, File, QueryResolvers} from "../generated/graphql";
+import {ClaimDto, ClaimOutcomeDto, ClaimStatusDto, FileDto} from "../api/upstreams/claimService";
 import {LocalizedStrings} from "../translations/LocalizedStrings";
-import {transformContract} from "./contracts";
+import {transformContract, transformTrialToFakeContract} from "./contracts";
 import {AWS_CLAIMS_S3_BUCKET} from "../config";
 import {s3} from "../api/s3";
 
 const THIRTY_MINUTES = 60 * 30
 
-const statusMap = {
-  [ClaimStatusDto.CREATED]: ClaimStatus.Submitted,
-  [ClaimStatusDto.IN_PROGRESS]: ClaimStatus.BeingHandled,
-  [ClaimStatusDto.CLOSED]: ClaimStatus.Closed,
-  [ClaimStatusDto.REOPENED]: ClaimStatus.Reopened,
+interface ClaimPrivate {
+  _contractId?: string
 }
 
-const outcomeMap = {
-  [ClaimOutcomeDto.PAID]: ClaimOutcome.Paid,
-  [ClaimOutcomeDto.NOT_COMPENSATED]: ClaimOutcome.NotCompensated,
-  [ClaimOutcomeDto.NOT_COVERED]: ClaimOutcome.NotCompensated,
+export const getContractByClaim: ClaimResolvers['contract'] = async (parent, _args, context) => {
+
+  // try find contract
+  const parentAsPrivate = (parent as ClaimPrivate)
+  if (!parentAsPrivate._contractId) return undefined
```

Comment:
```suggestion
  if (!parentAsPrivate._contractId) {
    return undefined
  }
```

`if`-statements without curly braces are error-prone: https://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/

The following form works if we want to keep it as a one-liner, but really we should just go with whatever formatting `prettier` considers to be correct 🙂 

```suggestion
  if (!parentAsPrivate._contractId) { return undefined }
```